p/scripts/main.js

```diff
@@ -1099,10 +1099,8 @@ function init_shortcuts() {
 
 			const link_go_website = document.querySelector('.flux.current a.go_website');
 			if (link_go_website) {
-				const newWindow = window.open();
+				const newWindow = window.open(link_go_website.href, '_blank', 'noopener');
 				if (newWindow) {
-					newWindow.opener = null;
-					newWindow.location = link_go_website.href;
 					ev.preventDefault();
```

Comment:
There's a regression hiding in here. newWindow is `null`, just as if it was blocked. That being said, it probably makes sense to do preventDefault without any conditions regardless? As a user I'm not expecting it to do whatever the default browser action is, but for it to be overridden as per the settings.

![image](https://github.com/user-attachments/assets/c6a7e957-b636-4ff7-b856-3e75f09b2e94)

Reply:
Do you plan to send a PR?

Reply:
https://github.com/FreshRSS/FreshRSS/pull/7089